### ISSUE TITLE
Shakespeare Summarizes Everything

### ISSUE THREAD

**Issue by jrladd** [NaNoGenMo/2017#118]
## [View Code Here](https://github.com/jrladd/shakes_summary)

What if Shakespeare wrote the backs of 19th-century book jackets? I matched randomized lines from [Shakespeare's works](http://www.folgerdigitaltexts.org/) with [Yellowback book covers](https://www.flickr.com/photos/yellowbacks/). A dead simple approach, but I'm pleased with the results.

To run the code, you'll need your own API key and API secret from Flickr. The Python script creates a markdown file that I converted to HTML and, with some adjustments and styles, posted [here](http://jrladd.com/shakes_summary).

This is inspired by Stephen Pentecost's [Mutable Stanzas](http://montaukedp.com/apps/mutableStanzas/) and Kate Beaton's [Gorey book covers](http://www.harkavagrant.com/index.php?id=242). I wanted to make a project that plays with the way our minds fill in the gaps between text and image, as well as between lines that weren't intended to be together. The result is nonsense that makes sense despite itself.

A sample (so I can earn that sweet "preview" label):

![sample](https://user-images.githubusercontent.com/6046784/33242220-68453c24-d29f-11e7-85aa-6682177e7a11.png)

**Comment by hugovk** [comment]
Looks great!

Did you get to 50k?

**Comment by jrladd** [comment]
Yes, I just didn’t post all of it. The page was already slow after 500 images. I wound up doing 1500. Is that okay?

**Comment by hugovk** [comment]
Maybe make a 50k one too along with the shorter one?

> The only rule is that you share at least one novel and also your source code at the end.

(where novel = 50k+ words, although we've had graphic novels which creatively interpret the graphic as taking some amount of words :)

**Comment by jrladd** [comment]
I won't argue for an exception in this case. The constraints are part of the fun, right? 50k MD file coming shortly.

**Comment by jrladd** [comment]
Done! [68,392 words by my count.](http://jrladd.com/shakes_summary/) I got in the spirit and put it all online too—speed issue is really not too bad.

**Comment by jrladd** [comment]
Also, other folks reading this might be interested to know: the lines have been shuffled but not randomly sampled. Therefore there are no repeating lines (or images for that matter) anywhere—each summary has a unique line set.
